### PR TITLE
IESG review comments

### PR DESCRIPTION
**Pull request by cdh4u**
Important nits; please address:

1) Sec 14.2: RFC 6928, RFC 5389, and RFC 6298 should be actual references and listed in the reference section.

2) Sec 14.2: Maybe you should really not use normative language in the reasoning part of this section?!
    I find especially this SHOULD in the following sentence confusing (as it doesn't give a reference to another RFC that defines this):
   "Let MinPacing be the minimum pacing interval between
          transactions, which SHOULD be 5ms."
   Given that the previous text says:
  "all transactions from all agents [...] MUST NOT be sent more often than once
   every 5ms"
  My recommendation: please use lower cases "shoulds" and "musts" in the reasoning part!

3) Sec 19.4.1: "(say, Ta=20ms)"
Maybe use also a value of 50ms here to avoid confusion.

### CODE REVIEW

File: draft-ietf-ice-rfc5245bis.xml [hunk 1]
```diff
@@ -1156,33 +1137,22 @@ different for two candidates allocated from different IP addresses,
...
 Next, an agent chooses a default candidate for each component of each
 data stream. If a host is IPv4-only, there would only be one
 candidate for each component of each data stream, and therefore that
-candidate is the default. If a host is IPv6 or dual-stack, the
-selection of default is a matter of local policy. This default SHOULD
-be chosen such that it is the candidate most likely to be used with
-a peer. For IPv6-only hosts, this would typically be a globally scoped
-IPv6 address. For dual-stack hosts, the IPv4 address is RECOMMENDED.
-</t>
+candidate is the default. If a host is IPv6-only hosts, the default candidate
```
Comment (kaduk) [inline draft-ietf-ice-rfc5245bis.xml]: The second "hosts" is unneeded.

File: draft-ietf-ice-rfc5245bis.xml [hunk 2]
```diff
@@ -1863,10 +1830,10 @@ to perform connectivity checks for, abort these steps.
 
 </list></t>
 
-<t>Once the agent has picked a candidate pair, for which a connectivity check is to
-be performed, the agent performs the check by sending a STUN request from the base associated
-with the local candidate of the pair to the remote candidate of the pair, as described in
-<xref target="sec-send-check"/>.
+<t>Once the agent has picked a candidate pair for which a connectivity check is to
+be performed, the agent starts a check, and sends the Binding request from the base
```
Comment (kaduk) [inline draft-ietf-ice-rfc5245bis.xml]: This  comma can be removed.

File: draft-ietf-ice-rfc5245bis.xml [hunk 3]
```diff
@@ -876,10 +850,15 @@ following exceptions:
...
-    <t>Host candidates corresponding to IPv6 link-local addresses MUST NOT 
-    be gathered.</t>
+    <t>If one or more host candidates corresponding to an IPv6 
+    address generated using a mechanism that prevents location 
+    tracking <xref target="RFC7721"/> are gathered, host candidates 
+    corresponding to IPv6 addresses that do allow location tracking 
+    that are configured on the same interface and are part of the 
+    same network prefix MUST NOT be gathered; and host candidates 
+    corresponding to IPv6 link-local addresses <xref target="RFC4291"/>
+    MUST NOT be gathered.</t>
```
Comment (kaduk) [inline draft-ietf-ice-rfc5245bis.xml:863]: Does this need to get run by the WG?

File: draft-ietf-ice-rfc5245bis.xml (hunk 1 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Will fix.

File: draft-ietf-ice-rfc5245bis.xml (hunk 2 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Will fix.

File: draft-ietf-ice-rfc5245bis.xml (hunk 3 above)
Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml:863]: I'll let the chairs deal with that, but this text was already in an earlier version of the draft.

File: draft-ietf-ice-rfc5245bis.xml [hunk 4]
```diff
@@ -3359,7 +3304,7 @@ implementations.
           |D=$L-PRIV-1   |              |              |
           |MA=$NAT-PUB-1 |              |              |
           |<-------------|              |              |
-          |Data flows    |              |              |
+          |              |              |              |
```
Comment (adamroach) [inline draft-ietf-ice-rfc5245bis.xml]: "Data Flows" is still accurate here for the majority of use cases, as the pair can be used as soon as it becomes "valid," and most implementations will be attempting to minimize setup delay.

Comment (cdh4u) [reply draft-ietf-ice-rfc5245bis.xml]: Seems like it got dropped for some reason. I'll put it back.

**Pull request merged**